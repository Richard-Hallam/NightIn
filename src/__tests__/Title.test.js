import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Title from '../components/title';

test('Title component renders correctly', () => {
  const { container } = render(<Title />);
  expect(container).toMatchSnapshot();
});