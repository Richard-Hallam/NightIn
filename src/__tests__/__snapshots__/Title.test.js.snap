// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component renders correctly 1`] = `
<div>
  <div
    class="title"
  >
    <h2>
      How about we...
      <span
        class="underline"
        id="cycleText"
      >
        WhittleIt?
      </span>
    </h2>
    <div>
      <div
        class="button-container"
      >
        <button
          class="side-button"
        >
          <span
            class="material-symbols-outlined"
            style="font-size: 100pt;"
          >
            person_add
          </span>
        </button>
        <button
          class="match-button"
        >
          let's watch.
        </button>
      </div>
    </div>
  </div>
</div>
`;
